default/stringtables/en.txt

```diff
@@ -18082,17 +18088,20 @@ Darquan
 Darrian
 Deadwood
 Deemi
+Delos
 Delphi
 Demeter
 Deneb
 Denebola
+Denobula
```

Comment:
almost a duplicate...

Reply:
just loop the levenshtejn or edit distance .. i'm pretty sure there are other close matches. besides, i seem to recall having added that first name many years beack, together with names like Leibowitz and Ingwaz and others where i can clearly reconstruct what was going through my head trying to think of more names … nevermind the credits, just glad its all still here ^^ and Even Happier: someone really did the greek alphabet thing to expand star names, great stuff!

Reply:
that there are other problems or questionable cases doesn't mean this should be left...